src/components/Application/ApplicationStart.tsx

```diff
@@ -60,23 +86,72 @@ const ApplicationStart: React.FC<ApplicationStartProps> = (props) => {
           <Segment basic>
             <Header as="h5">{strings.SUBTITLE_APPLICATION_STEPS}</Header>
             <Header as="h5">{strings.TITLE_STEPS.toUpperCase()}</Header>
-            <List divided relaxed>
-              {sections &&
-                sections.map((section) => (
-                  <List.Item key={`list-item-${section.code}`}>
-                    <List.Icon name="circle outline" />
-                    <List.Content>{section.title}</List.Content>
-                  </List.Item>
-                ))}
+            <List divided relaxed="very">
+              {sectionsProgress &&
+                Object.entries(sectionsProgress).map(([_, { info, progress, link }]) => {
+                  return (
+                    <List.Item
+                      key={`list-item-${info.code}`}
+                      children={
+                        <Grid>
+                          <Grid.Column width={1}>
+                            {progress?.completed ? (
+                              <List.Icon color="green" name="check circle" />
+                            ) : (
+                              <List.Icon name="circle outline" />
+                            )}
+                          </Grid.Column>
+                          <Grid.Column width={10}>
+                            <p>{info.title}</p>
+                          </Grid.Column>
+                          <Grid.Column width={3}>
+                            {progress && progress.done > 0 && progress.total > 0 && (
+                              <Progress
+                                percent={(100 * progress.done) / progress.total}
+                                size="tiny"
+                                success={progress.valid}
+                                error={!progress.valid}
+                              />
+                            )}
+                          </Grid.Column>
+                          <Grid.Column width={2}>
+                            {progress && info.code === firstIncomplete && (
+                              <Button
+                                color="blue"
+                                as={Link}
+                                to={link} // TODO: Link not working - page doesn't refresh!
```

Comment:
Hmmm, very strange. Would be good to get to the bottom of this in this PR if possible.

Reply:
Finally!!! Done: c7bd60a